DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -14145,8 +14145,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/duckduckgo/BrowserServicesKit";
 			requirement = {
-				kind = exactVersion;
-				version = 193.0.1;
+				kind = revision;
+				revision = e2545b64853587b28dc850f0f3be3e621fd18a6f;
```

Comment:
Looks like there's [already a BSK release](https://github.com/duckduckgo/BrowserServicesKit/releases/tag/193.0.2) to update to the version that macOS needs an adjustment for, we can set this to point at `192.0.2` instead.